content/en/user-guide/tools/lambda-tools/hot-reloading/index.md

```diff
@@ -97,46 +115,24 @@ $ cd /tmp
 $ git clone git@github.com:awsdocs/aws-doc-sdk-examples.git
 {{< / command >}}
 
-#### Starting up LocalStack
-
-First, we need to make sure we start LocalStack with the right configuration.
-This is as simple as setting `LAMBDA_REMOTE_DOCKER`(see the [Configuration Documentation]({{< ref "configuration#lambda" >}}) for more information):
-
-{{< command >}}
-$ LAMBDA_REMOTE_DOCKER=0 localstack start
-{{< / command >}}
-
-Accordingly, if you are launching LocalStack via Docker or Docker Compose:
-
-```yaml
-#docker-compose.yml
-
-services:
-  localstack:
-    ...
-    environment:
-      ...
-      - LAMBDA_REMOTE_DOCKER=false
-```
-
 #### Creating the Lambda Function
 
 To create the Lambda function, you just need to take care of two things:
-1. Deploy via an S3 Bucket. You need to use the magic variable `__local__` as the bucket.
+1. Deploy via an S3 Bucket. You need to use the magic variable `hot-reload` as the bucket.
 2. Set the S3 key to the path of the directory your lambda function resides in.
    The handler is then referenced by the filename of your lambda code and the function in that code that needs to be invoked.
 
 So, using the AWS example, this would be:
 
 {{< command >}}
 $ awslocal lambda create-function --function-name my-cool-local-function \
-    --code S3Bucket="__local__",S3Key="/tmp/aws-doc-sdk-examples/python/example_code/lambda/boto_client_examples" \
+    --code S3Bucket="hot-reload",S3Key="/tmp/aws-doc-sdk-examples/python/example_code/lambda/boto_client_examples" \
     --handler lambda_handler_basic.lambda_handler \
     --runtime python3.8 \
-    --role cool-stacklifter
+    --role arn:aws:iam::000000000000:role/lambda-role
```

Comment:
Should we keep the sample ARN region-agnostic or add `us-east-1`?

(I generally prefer `123456789012` to clarify the number of digits but 12x `0` seems more consistent with LocalStack returning it and assuming it as role)

Reply:
IAM is account-global without region. => Correct as now